src/java/picard/vcf/VcfFormatConverter.java

```diff
@@ -53,8 +53,17 @@
  * @author jgentry@broadinstitute.org
  */
 @CommandLineProgramProperties(
-        usage = "Convert a VCF file to a BCF file, or BCF to VCF.\n" +
-                "Input and output formats are determined by file extension.",
+        usage = "Convert a VCF file to a BCF file, or BCF to VCF." +
+                "Input and output formats are determined by file extension." +
```

Comment:
" by file extension" -> "based on the file extensions specified in the file names"